docs/presets/jetpack.md

```diff
@@ -0,0 +1,25 @@
+[Return to Presets](./README.md#jetpack)
+
+# Jetpack Preset
+
+This is a pseudo minimalist preset inspired by the [geometry](https://github.com/geometry-zsh/geometry) and [spaceship](https://github.com/spaceship-prompt/spaceship-prompt) prompts.
+
+> Jetpack uses the terminal's color theme.
+
+![Screenshot of Jetpack preset](/presets/img/jetpack.png)
+
+### Prerequisite
+
+- [Jetbrains Mono](https://www.jetbrains.com/lp/mono/) installed and enabled in the terminal.
+
+### Manual Configuration
+
+[Click to download TOML](/presets/toml/jetpack.toml)
+
+### Configuration
+
+```sh
+starship preset jetpack-o ~/.config/starship.toml
```

Comment:
There is a typo in this line: you forgot a space between `… jetpack` and `-o`. It should be `starship preset jetpack -o ~/.config/starship.toml`.